idl/chromadb/proto/chroma.proto

```diff
@@ -93,6 +93,11 @@ message OperationRecord {
     Operation operation = 4;
 }
 
+message RequestVersionContext {
+    uint32 collection_version = 1;
+    uint64 log_position = 2;
```

Comment:
nit: the type is int64 elsewhere. Here it is uint64. uint64 makes more sense since they are all >= 0

Reply:
Yeah I made a todo in the code to go make it uint64. I do not know why its int64 elsewhere.